modules/network.c

```diff
@@ -99,22 +100,16 @@ void scan_statistics(gboolean reload)
             __statistics = h_strdup_cprintf("[%s]\n",
                                             __statistics,
                                             tmp);
-
             g_free(tmp);
-          } else if (isdigit(buffer[4])) {
-            gchar *tmp1 = buffer + 4,
-                  *tmp2 = tmp1;
 
-            while (*tmp2 && !isspace(*tmp2)) tmp2++;
-            *tmp2 = 0;
-            tmp2++;
+          } else {
+            gchar *tmp = buffer;
 
-            *tmp2 = toupper(*tmp2);
+            while (*tmp && isspace(*tmp)) tmp++;
 
-            __statistics = h_strdup_cprintf("%s=%s\n",
+            __statistics = h_strdup_cprintf("<b> </b>#%d=%s\n",
```

Comment:
It's a hack, but it works. The keys must be unique for each line, and everything after # is hidden. Something must be before the # or the key is "" after the spaces are stripped and it won't be displayed. A bolded space won't be stripped.